iot-hub/Samples/service/ImportExportDevicesSample/ImportExportDevicesSample.cs

```diff
@@ -273,8 +273,8 @@ private async Task GenerateAndAddDevices(string hubConnectionString, string cont
                     {
                         SymmetricKey = new SymmetricKey
                         {
-                            PrimaryKey = CryptoKeyGenerator.GenerateKey(32),
-                            SecondaryKey = CryptoKeyGenerator.GenerateKey(32),
+                            PrimaryKey = GenerateKey(32),
```

Comment:
We've deprecateds the method previous used in the SDK, so I added a helper method below to do this.